Extended Project 3/lib/logic_methods_rspec.rb

```diff
@@ -0,0 +1,25 @@
+require_relative 'logic_methods'
+
+RSpec.describe "Test logic_methods" do
+
+  #True or false if file *.txt exists or not in Data file
+  
+  it "should return TRUE because Carlos.txt exists" do
+    result = check_file_data_existence("Carlos") #txt is 
+    expect(result).to be true
+  end
+  it "should return FALSE because Someone.txt doesn't exist" do
+    result = check_file_data_existence("Someone") #txt is 
+    expect(result).to be false
+    
+  end
+ # Verify read_string is returning a file
+
+  it "should webpage about.html must be expect when is called" do
+      result=read_string("about.html")
+      # about. html - webpage where Members' answers are shown.
+      expect(result).to include(File.read("../about.html"))
+      
+  end
+
+end
```

Comment:
Congrats for explain your code, but in this case, is not necessary because your test is auto describe itself, maybe could you gather better test like this:
```ruby
describe "True or false if file *.txt exists or not in Data file"  do
  #Your test's
end
describe "Verify read_string is returning a file"  do
  #Your test's
end
```

Reply:
In order to make it for dummies, I prefer to make as much clear as I can be.